### ISSUE TITLE
Tons of posts on one issue

### ISSUE THREAD

**Issue by meneal** [CSC510-2015-Axitron/project2#10]
We were guilty of this for sure.  If there are a huge number of posts on one issue it means that the issue should have been broken up into smaller issues.  At least that's what I would think.  Either that or it's just a really terrible problem, in which case it's still not a great smell.  You could spin this the opposite way and say that if there are tons of posts on issues it is a good smell since there is good discussion on each individual issue.

**Comment by duh102** [comment]
Returns issueID and the number of comments on that issue:

``` SQL
select issueID, count(*) from comment group by issueID;
```

**Comment by duh102** [comment]
![boxplot_comments_on_issues](https://cloud.githubusercontent.com/assets/6590396/7281800/d76cadc2-e8f9-11e4-8328-0475e96dd5d8.png)

**Comment by meneal** [comment]
We're group nine aren't we?

**Comment by duh102** [comment]
Yep, we were much more talkative than the other groups it seems like.

**Comment by meneal** [comment]
I'm biased, but I think we could call that a good smell actually.  :)
